src/boot/efi/stub.c

```diff
@@ -343,22 +352,78 @@ static EFI_STATUS cmdline_append_and_measure_addons(
                         continue;
                 }
 
-                _cleanup_free_ char16_t *tmp = TAKE_PTR(buffer),
+                /* Extract initrd if available */
+                if (szs[UNIFIED_SECTION_INITRD] > 0) {
+                        bool m = false;
+                        additions->initrd_size = szs[UNIFIED_SECTION_INITRD];
+                        additions->initrd_base_override =
+                                POINTER_TO_PHYSICAL_ADDRESS(loaded_addon->ImageBase) +
+                                addrs[UNIFIED_SECTION_INITRD];
+
+                        (void) tpm_log_event_ascii(
+                                TPM_PCR_INDEX_INITRD_ADDONS,
+                                POINTER_TO_PHYSICAL_ADDRESS(unified_sections[UNIFIED_SECTION_INITRD]),
+                                strsize8(unified_sections[UNIFIED_SECTION_INITRD]),
+                                unified_sections[UNIFIED_SECTION_INITRD],
+                                &m);
```

Comment:
hmm?

Reply:
Yeah I was mindlessly replicating the measurement mechanism for unified sections, but actually, we don't need section names here

Reply:
removed